### DISCUSSION TITLE
List of Colors

### DISCUSSION THREAD

**Discussion by Witchdoc59** [Bodmer/TFT_eSPI#2935] [Q&A - General]
In the Free_Font_Demo there are lines like this; tft.setTextColor(TFT_YELLOW, TFT_BLACK);

Is there a list of the colors that we can use?

Is there a reference for the keywords?  for example the line  tft.setTextColor(TFT_YELLOW, TFT_BLACK);  setTextColor takes two arguments.  What are the arguments and what values can we use.

All and any comments or suggestions greatly appreciated.

**Comment by Bodmer** [comment]
A list of available colours is [here](https://github.com/Bodmer/TFT_eSPI/blob/master/TFT_eSPI.h#L302).

Functions descriptions are [here](https://github.com/Bodmer/TFT_eSPI/blob/master/TFT_eSPI.h#L436-L820).

One day this info will be put into some documentation, in the meantime most of the available functions are described in the examples.
